packages/clerk-js/src/ui/components/CreateOrganization/CreateOrganizationForm.tsx

```diff
@@ -121,6 +123,31 @@ export const CreateOrganizationForm = (props: CreateOrganizationFormProps) => {
             organization={{ name: nameField.value }}
             onAvatarChange={async file => await setFile(file)}
             onAvatarRemove={file ? onAvatarRemove : null}
+            avatarPreviewPlaceholder={
+              <IconButton
+                variant='ghost'
+                colorScheme='neutral'
+                aria-label='Upload image'
+                icon={
+                  <Icon
+                    size='md'
+                    icon={isHovered ? Upload : Image}
+                  />
+                }
+                sx={theme => ({
+                  width: theme.sizes.$11,
+                  height: theme.sizes.$11,
+                  borderRadius: theme.radii.$md,
+                  backgroundColor: theme.colors.$avatarBackground,
+                  ':hover': {
+                    backgroundColor: theme.colors.$avatarBackground,
+                    opacity: 0.8,
+                  },
+                })}
+                onMouseEnter={() => setIsHovered(true)}
+                onMouseLeave={() => setIsHovered(false)}
```

Comment:
Nice touch,

What i would do is to only keep one of them (pref. the Upload icon) and just have an "zoom in" animation on hover.


```tsx
<Icon
  size='md'
  icon={Upload}
  sx={t => ({
     ':hover': {
        transform: 'scale(1.2)',
     },
  })}
/>
```